src/components/ActionSheetAwareScrollView/__mocks__/index.android.tsx

```diff
@@ -0,0 +1,11 @@
+import {Actions, ActionSheetAwareScrollViewContext, ActionSheetAwareScrollViewProvider} from './ActionSheetAwareScrollViewContext';
```

Comment:
`/components/ActionSheetAwareScrollView/__mocks__/index.android.tsx`, `/components/ActionSheetAwareScrollView/__mocks__/index.ios.tsx`, and `/components/ActionSheetAwareScrollView/__mocks__/index.tsx` are the same. Combine them into a single `/components/ActionSheetAwareScrollView/__mocks__/index.tsx`